src/expectation.coffee

```diff
@@ -74,7 +74,19 @@ module.exports = class Expectation
 
     @manager._checkPageSettings(loader.originalOptions)
 
-    _.all ['include', 'only', 'order', 'filters', 'perPage', 'page', 'limit', 'offset', 'search'], (optionType) =>
+    optionKeys = ['include'
+                  'only'
+                  'order'
+                  'filters'
+                  'perPage'
+                  'page'
+                  'limit'
+                  'offset'
+                  'search'
+                  'cacheKey'
```

Comment:
We had to extract this out into an array and pass that in, otherwise we would go over the 144 character line limit. It was done to fix a linting error. We couldn't figure out in Coffeescript how to allow for multiple line parameters.